FIDO2_make_auth_data.md

```diff
@@ -0,0 +1,32 @@
+## FIDO2 Make Authenticator Data 
```

Comment:
this file isn't referenced by anything and seems to be obsolete from what I see.

Reply:
Ditto